src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -183,6 +183,21 @@ public void ReportAaptErrorsInOriginalFileName ()
 			}
 		}
 
+		[Test]
```

Comment:
This test should be redundant with the addition to `AndroidRegExTests.cs`. The only things this will do is slow down the tests (MSBuild-based tests are *slow*), and trigger a warning if/when `aapt` changes the error message it generates.

@dellis1972: agreed?

Reply:
We should extend the `AndroidRegExTests.cs` test cases rather than adding new tests which do a full build I think.

Reply:
actually. I take that back.. the `AndroidRegExTests.cs` only tests the regex.. it doesn't test the logic of what happens after we get the regex.. So we need this test.